nerfstudio/fields/vanilla_nerf_field.py

```diff
@@ -89,11 +93,15 @@ def __init__(
     def get_density(self, ray_samples: RaySamples):
         if self.use_integrated_encoding:
             gaussian_samples = ray_samples.frustums.get_gaussian_blob()
+            if self.temporal_distortion is not None:
+                positions = self.temporal_distortion(gaussian_samples, ray_samples.times)
```

Comment:
This will break because the `guassian_samples` are means and covariances, not just positions.